src/components/accordion.ts

```diff
@@ -0,0 +1,78 @@
+import {Component, View, NgClass, OnDestroy} from 'angular2/angular2';
+
+@Component({
+  selector: 'ngb-accordion, [ngb-accordion]',
+  properties: ['onlyOneOpen: closeOthers'],
+  host: {'class': 'panel-group'}
```

Comment:
I think it's actually unnecessary to even have that `panel-group` class in BS4 - it also is not useful in BS3 unless the element is a block element. Might be worth considering putting a wrapper div in the template with that class on it instead.